package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sanctuary-benchmark",
+  "description": "Basis for easy benchmarking and sharing of the results",
+  "version": "0.0.0",
+  "main": "index.js",
```

Comment:
I like to make `"files"` explicit. Having to consult the npm documentation in order to know what's included by default is a bit of a nuisance.

It may seem contradictory to advocate explicit `"files"` but implicit `"main"`, but the rules for the latter are much simpler and I believe every project should use the default entry point (we can `require` to our heart's content, after all).